### PR TITLE
Fix: sensitive environment variables alias

### PR DESCRIPTION
**Pull request by yaronya**
### Issue
`command-line-args` package only supports aliases as one character, while sensitive environment variables where set to `-vs`

### Solution
Changed it to `-q`

### CODE REVIEW

File: node/env0-deploy-cli.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const optionDefinitions = [
   { name: 'blueprintId', alias: 'b', type: String, description: 'The Blueprint id you would like to deploy' },
   { name: 'environmentName', alias: 'e', type: String, description: 'The environment name you would like to create, if it exists it will deploy to that environment' },
   { name: 'environmentVariables', alias: 'v', type: String, multiple: true, defaultValue: [], description: 'The environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"' },
-  { name: 'sensitiveEnvironmentVariables', alias: 'vs', type: String, multiple: true, defaultValue: [], description: 'The sensitive environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"' },
+  { name: 'sensitiveEnvironmentVariables', alias: 'q', type: String, multiple: true, defaultValue: [], description: 'The sensitive environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"' },
```
Comment (omryhay1) [inline node/env0-deploy-cli.js:14]: q? 
anyway please also update the README file,

Comment (omryhay1) [reply node/env0-deploy-cli.js:14]: While you're at it please remove the `,` and the newline here - https://github.com/env0/env0-client-integrations/blob/master/node/tests/env0-deploy-cli.spec.js#L27

Comment (avnerenv0) [reply node/env0-deploy-cli.js:14]: can we delete that part of the readme? 
that's just duplication and we can point the user to `node env0-cli --help` or whatever shows him the help

Comment (omryhay1) [reply node/env0-deploy-cli.js:14]: I don't mind removing the readme but it should be aligned.

Comment (yaronya) [reply node/env0-deploy-cli.js:14]: > q?

@omryhay1 `-s` was already taken by API SECRET 🤷

**Pull request merged**